app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -185,28 +189,32 @@ class SharedCallingViewModel @Inject constructor(
 
     fun toggleMute() {
         viewModelScope.launch {
-            callState = if (callState.isMuted) {
-                callState.copy(isMuted = false)
-            } else {
-                callState.copy(isMuted = true)
+            callState.isMuted?.let {
```

Comment:
if the default value is `null` would there be any edge case that it will still remain `null` and we will not be able to toggle the mute/unmute button?

Reply:
No because we are already setting its value at launch time in `observeCallState()`, same thing for isCameraOn.